core/src/pixelcolor/gray_color.rs

```diff
@@ -8,10 +8,13 @@ pub trait GrayColor: PixelColor {
     /// Returns the luma channel value.
     fn luma(&self) -> u8;
 
-    /// Black color (0% luma)
+    /// Black color (0% luma).
     const BLACK: Self;
 
-    /// White color (100% luma)
+    /// Middle gray color (50% luma).
+    const GRAY_50: Self;
```

Comment:
This might be worth an `Added` changelog entry, unless you think it's too small to bother with?

Reply:
I've removed the constant again. `GRAY_50` wasn't really at exactly 50%, because each color has an even number of possible luma values. And this change would be a breaking API change, because the `GrayColor` trait isn't sealed and could theoretically be implemented externally (very unlikely but possible).